sc/social/like/tests/test_upgrades.py

```diff
@@ -285,3 +286,39 @@ def test_enable_social_media_behavior(self):
                 continue  # not a Dexterity-based content type
             behaviors = list(fti.behaviors)
             self.assertIn(ISocialMedia.__identifier__, behaviors)
+
+
+class To3046TestCase(UpgradeTestCaseBase):
+
+    def setUp(self):
+        UpgradeTestCaseBase.setUp(self, u'3045', u'3046')
+
+    def test_upgrade_to_3046_registrations(self):
+        version = self.setup.getLastVersionForProfile(self.profile_id)[0]
+        self.assertGreaterEqual(int(version), int(self.to_version))
```

Comment:
I already found the issue and fixed it: https://github.com/collective/sc.social.like/pull/118/commits/78bc1e17e1e99c7a4c833a3d1c2600bb232321f3